sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Customized/Models/TimeSeriesInstance.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// Time series instances are the time series themselves. In most cases, the deviceId or assetId is the
+    /// unique identifier of the asset in the environment. Instances have descriptive information associated
+    /// with them called instance fields. At a minimum, instance fields include hierarchy information. They
+    /// can also include useful, descriptive data like the manufacturer, operator, or the last service date.
+    /// </summary>
+    [CodeGenModel("TimeSeriesInstance")]
+    [CodeGenSuppress("TimeSeriesInstance", typeof(IEnumerable<object>), typeof(string))]
+    [CodeGenSuppress(
+        "TimeSeriesInstance",
+        typeof(IList<object>),
+        typeof(string),
+        typeof(string),
+        typeof(string),
+        typeof(IList<string>),
+        typeof(IDictionary<string, object>))]
+    public partial class TimeSeriesInstance
+    {
+        [CodeGenMember("TimeSeriesId")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage(
+            "CodeQuality",
+            "IDE0051:Remove unused private members",
+            Justification = "Autorest does not support changing type for proprties. In order to turn TimeSeriesId from" +
+            "a list of objects to a strognly typed object, TimeSeriesId has been renamed to TimeSeriesIdInternal and a" +
+            "new property, TimeSeriesId, has been created with the proper type.")]
+        private IList<object> TimeSeriesIdInternal { get; }
+
+        /// <summary>
+        /// Time Series Id that uniquely identifies the instance. It matches Time Series Id properties in an environment.
+        /// </summary>
+        public ITimeSeriesId TimeSeriesId { get; }
```

Comment:
what is the reasoning for making the TimeSeriesId and interface ?

Reply:
In the next iteration of this PR, this file will no longer exist.